internal/runners/branch/tree.go

```diff
@@ -17,108 +16,87 @@ type branchNode struct {
 
 type tree map[branchNode]tree
 
-type BranchTree struct {
-	tree                  tree
-	branches              mono_models.Branches
-	rootBranches          mono_models.Branches
-	localBranch           string
-	branchFormatting      string
-	localBranchFormatting string
+type BranchOutput struct {
+	branches    mono_models.Branches
+	localBranch string
 }
 
 const (
-	edgeLink string = "│  "
-	edgeMid  string = "├─"
-	edgeEnd  string = "└─"
-)
+	prefixLink string = "│  "
+	prefixMid  string = "├─"
+	prefixEnd  string = "└─"
 
-func NewBranchTree() *BranchTree {
-	return &BranchTree{tree: make(tree)}
-}
+	branchFormatting      string = "[NOTICE]%s[/RESET]"
+	localBranchFormatting string = "[ACTIONABLE]%s[/RESET] [DISABLED](Current)[/RESET]"
+)
 
-func (bt *BranchTree) BuildFromBranches(branches mono_models.Branches) {
-	bt.rootBranches = getRootBranches(branches)
-	for _, branch := range bt.rootBranches {
-		bt.tree[branchNode{branch.Label, branch.BranchID.String()}] = buildBranchTree(branch, branches)
+func NewBranchOutput(branches mono_models.Branches, localBranch string) *BranchOutput {
+	return &BranchOutput{
+		branches:    branches,
+		localBranch: localBranch,
 	}
 }
 
-func buildBranchTree(currentBranch *mono_models.Branch, branches mono_models.Branches) tree {
-	t := getChildren(currentBranch, branches)
-	for _, branch := range branches {
-		// Discard any branches without tracking information as we are only interested
-		// in child branches of the current branch
-		if branch.Tracks == nil {
-			continue
-		}
-
-		// Check that this branch is a child branch and recursively build its tree
-		if _, ok := t[branchNode{branch.Label, branch.BranchID.String()}]; ok {
-			t[branchNode{branch.Label, branch.BranchID.String()}] = buildBranchTree(branch, branches)
-		}
+func (bt *BranchOutput) MarshalOutput(format output.Format) interface{} {
+	if format != output.PlainFormatName {
+		return branchListing(bt.branches, bt.localBranch)
```

Comment:
```suggestion
		return bt.branches
```

You can get rid of `branchListing`.